### PR TITLE
Adds explanation to readme of how to use the javascript library

### PR DESCRIPTION
**Pull request by NellWaliczek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -431,6 +431,97 @@ Certain WebVR `Gamepad` objects have some components with an inverted `yAxis` ca
...
+      const weightedVisualizations = xrComponent.getWeightedVisualizations();
+      MyEngine.updateInputVisuals(key, weightedVisualizations);
+    });
+  });
+}
+```
+
+> TODO Add an explanation for how to updated the visuals based on the weighted visualizations
+
+## Target Ray Origin
```
Comment (toji) [inline README.md]: So this is feedback that's a couple of PRs too late, but it didn't really click with me till I saw it used in context. The use of "origin" here feels to me like it only implies a position offset, so it read kind of weird to me at first. Is there a better term that we could use?

File: README.md [hunk 2]
```diff
@@ -431,6 +431,97 @@ Certain WebVR `Gamepad` objects have some components with an inverted `yAxis` ca
...
+In WebXR, the `XRInputSource.targetRaySpace` should be passed into `XRFrame.getPose()` to determined the pointing origin.
+
+```js
+const targetRayOrigin = xrFrame.getPose(xrInputSource.targetRaySpace, xrReferenceSpace);
+```
+
+### WebVR Target Ray Origin
+In WebVR there is no implicit mechanism for retrieving a target ray origin.  Instead, it must be retrieved from the the mapping via the `XRGamepad`
+```js
+const targetRayOrigin = xrGamepad.targetRayOrigin;
```
Comment (toji) [inline README.md]: Hm... maybe I just misread this previously: I thought that `targetRayOrigin` was a static offset from the base gamepad pose? That would appear to not be the case here if this is meant to be ~equivalent to the WebXR code above.

File: README.md (hunk 1 above)
Comment (NellWaliczek) [reply README.md]: I'm open to whatever, though I'd like to not hold this PR for it...

File: README.md (hunk 2 above)
Comment (NellWaliczek) [reply README.md]: Gah.  No that's me moving too fast.  I'll get an update pushed in a moment.

**Pull request merged**